libdnf/rpm/package.cpp

```diff
@@ -42,37 +42,37 @@ std::string Package::get_name() const {
     return cstring2string(solv::get_name(pool, id));
 }
 
-const char * Package::get_epoch_cstring() {
+const char * Package::get_epoch_cstring() const {
     Pool * pool = sack->pImpl->pool;
     return solv::get_epoch_cstring(pool, id);
 }
 
-unsigned long Package::get_epoch() {
+std::string Package::get_epoch() const {
     Pool * pool = sack->pImpl->pool;
-    return solv::get_epoch(pool, id);
+    return solv::get_epoch_cstring(pool, id);
```

Comment:
What about to make additional change? 
solv::get_epoch(pool, id) could return `char *`(like get_epoch_cstring). 
solv::get_epoch_cstring(pool, id); could be removed?
@dmach, what do you think?

Reply:
I'll take a look at how it works performance-wise.
If the performance is better or close to the current code, then I'll use the template functions.